pkg/source/openstack/client.go

```diff
@@ -54,6 +54,15 @@ type ExtendedVolume struct {
 	VolumeImageMetadata map[string]string `json:"volume_image_metadata,omitempty"`
 }
 
+// ExtendedServer The original `Server` structure does not contain the `Description` field.
```

Comment:
I think this is wrong. This and other code in your PR is from https://github.com/harvester/vm-import-controller/pull/40.

Reply:
🤦 let me fix it